pkg/agent/flowexporter/connections/conntrack_connections.go

```diff
@@ -231,20 +255,33 @@ func (cs *ConntrackConnectionStore) AddOrUpdateConn(conn *flowexporter.Connectio
 			conn.StopTime = time.Now()
 		}
 		metrics.TotalAntreaConnectionsInConnTrackTable.Inc()
+		// Add the connection to the priority queue.
+		pqItem := &flowexporter.ItemToExpire{
+			ConnKey:          connKey,
+			IsActive:         true,
+			ActiveExpireTime: currTime.Add(cs.activeFlowTimeout),
+			IdleExpireTime:   currTime.Add(cs.idleFlowTimeout),
+			IsDenyConn:       false,
+		}
+		conn.PriorityQueueItem = pqItem
+		pqItem.Conn = *conn
+		cs.expirePriorityQueue.Mutex.Lock()
+		heap.Push(cs.expirePriorityQueue, pqItem)
+		cs.expirePriorityQueue.Mutex.Unlock()
```

Comment:
This seems to be common code for both conntrack and deny connection store. We can have a method in connection store struct. "defer unlock" can be used.